app/src/utils/recordRequest.ts

```diff
@@ -234,6 +234,12 @@ async function createTags(projectId: string, loggedCallId: string, tags: Record<
 
   await kysely.insertInto("LoggedCallTag").values(tagsToCreate).execute();
 
+  const tagNames = tagsToCreate.map((tag) => tag.name);
+
+  await recordTagNames(projectId, tagNames);
```

Comment:
Can we just do `void recordTagNames(projectId, tagNames).catch(e => captureException(e)` to avoid the latency hit?

Reply:
Good idea